docs/documentation/platform/kms.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: "Key Management Service (KMS)"
+sidebarTitle: "Key Management (KMS)"
+description: "Learn how to manage and use cryptographic keys with Infisical."
+---
+
+## Introduction
+
+Infisical's <strong>Key Management System (KMS)</strong> allows you to create, store and manage cryptographic keys.
```

Comment:
Let's try to follow the format used for Internal PKI [here](https://infisical.com/docs/documentation/platform/pki/private-ca) split into **Concept** and **Workflow**:

- Concept: Should convey to users **concisely** what the service/feature is and why it's useful for them; essentially your introduction. (What / Why)
- Workflow: Assuming users understand the above concept, then now put yourself in their shoes and walk them through high-level steps of what to expect from working with this service/feature (How).

In this case, the workflow would consist of 3 steps:
1. Create KMS key.
2. Encrypt data using the KMS key.
3. Decrypt data using the KMS key.

Lastly, I think it would be super helpful to complement the concept section with a simple mermaid diagram (see PKI for example); I envision this to be some sequence diagram like the one below where secret key is replaced with Infisical KMS (feel free to improvise based on what you think is easiest to digest/understand for users):

<img width="790" alt="Screenshot 2024-10-03 at 8 03 31 AM" src="https://github.com/user-attachments/assets/9f83311c-4b7f-41fe-a9ed-aed91c5f2ab6">